addons/intercom_phone/functions/fnc_getIntercomChannel.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_fnc_getIntercomChannel
+
+  Author: Arend(Saborknight)
+    Gets the intercom channel in use by the crew of a vehicle,
+    in the order of: Driver, Gunner, Commander, Turrets, Cargo
+
+  Arguments:
+    0: Vehicle object <OBJECT>
+
+  Return Value:
+    Intercom Channel <NUMBER>
+
+  Example:
+    [_vehicle, _player] call TFAR_fnc_getIntercomChannel;
+
+  Public: Yes
+*/
+
+params ["_vehicle"];
+
+// Get any occupants, from crew down to passengers
+private _crew = crew _vehicle;
+_vehicle getVariable [format ["TFAR_IntercomSlot_%1",(netId (_crew select 0))],
```

Comment:
Is this variable set locally? I thought it was local only to remember what channel the player was last in when he left the vehicle.

Reply:
> Do you mean that the variable for that crew member is local to that crew member's machine only?

Yes, atleast it doesn't make any sense for other players to know that, its only used for local cache afaik

Reply:
Eh.... Let me get back to you on this after some testing with my guys.

Reply:
Seems to work fine